test/cpuinfo_x86_test.cc

```diff
@@ -219,6 +219,39 @@ TEST_F(CpuidX86Test, SkyLake) {
   EXPECT_EQ(GetX86Microarchitecture(&info), X86Microarchitecture::INTEL_SKL);
 }
 
+TEST_F(CpuidX86Test, SkyLakeXeon) {
+  SCOPED_TRACE("SkylakeXeon");
+  cpu().SetOsBackupsExtendedRegisters(true);
```

Comment:
could you please remove this field? Since you are only testing the microarchitecture here

Reply:
done